facefusion/face_helper.py

```diff
@@ -34,7 +34,7 @@ def warp_face(temp_frame : Frame, kps : Kps, template : Template, size : Size) -
 	return crop_frame, affine_matrix
 
 
-def paste_back(temp_frame : Frame, crop_frame : Frame, affine_matrix : Matrix) -> Frame:
+def paste_back_old(temp_frame : Frame, crop_frame : Frame, affine_matrix : Matrix) -> Frame:
```

Comment:
There is no need to keep the old code, it's still in the git history - please remove this.